src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProviderTest.java

```diff
@@ -61,9 +61,9 @@ public void calculateAmountDetailsWithPartAdmissions() {
             .build();
 
         AmountContent amountContent = amountContentProvider.create(claim);
-        assertThat(amountContent.getAdmittedAmount()).isEqualTo("£120.99");
-        assertThat(amountContent.getSubTotalAmount()).isEqualTo("£160.99");
-        assertThat(amountContent.getRemainingAmount()).isEqualTo("£160.99");
+        assertThat(amountContent.getAdmittedAmount()).isEqualTo("£40.99");
```

Comment:
why do these need to be changed?

Reply:
Good point. It isn't used in the PDF in _this_ case, but I'll make sure it's correct anyway in case it's used in others.